docs/gen-ai/azure-ai-inference.md

```diff
@@ -0,0 +1,83 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Azure AI Inference
+--->
+
+# Semantic Conventions for Azure AI Inference
+
+**Status**: [Experimental][DocumentStatus]
+
+The Semantic Conventions for [Azure AI Inference](https://learn.microsoft.com/azure/ai-studio) extend and override the [GenAI Semantic Conventions](README.md).
+
+## Azure AI Inference
+
+`gen_ai.system` MUST be set to `"az.ai.inference"` and SHOULD be provided **at span creation time**.
```

Comment:
@truptiparkar7 could you please bless this name? Thanks!

Reply:
How about `azureai`?

Reply:
we had some internal discussions on this and it seems we'll have other Azure AI things, so we need a way to distinguish. 

Also all the libs are called `azure-ai-inference` - https://learn.microsoft.com/azure/ai-studio/reference/reference-model-inference-api

So the motivation for this name was:
1. We have other services that will report `*.system` attributes. E.g. `az.search` could be reported in `db.system`. I.e. `az` or `azure` is a namespace
2. We already have `az.` attributes that are stable from Azure SDK side, so it should be `az`
3. we expect other things in `az.ai`, so we need `az.ai.inference` for this one

I'm totally open to suggestions from the product team.

Reply:
Yes I think it will be good to be consistent with the libraries this will be used with. So the name looks good to me.

Reply:
Overall this PR looks great and will be very helpful for us.